package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "base64-arraybuffer": "^0.1.5",
+    "es6-promisify": "^6.0.0",
```

Comment:
Why this vs. the builtin `util.promisify`?

Reply:
As I mentioned previously, for some reason parcel is not able to bundle `utils` module from node.

> The culprit was parcel not being able to bundle node's util library. So instead I opted for an equivalent npm package.